src/components/pill/pill.style.js

```diff
@@ -11,7 +11,7 @@ function addStyleToPillIcon(padding, fontSize, margin) {
   return `
     ${StyledIcon} {
       padding: ${padding};
-      margin-top: ${margin};
+      margin-top: ${margin !== undefined ? `${margin}` : '2px'};
```

Comment:
This is a change to a component so it would require a `fix` commit for this change only.